Modules/Sources/PointOfSale/Presentation/Settings/POSSettingsHardwareDetailView.swift

```diff
@@ -141,75 +135,6 @@ private extension POSSettingsHardwareDetailView {
         .navigationViewStyle(.stack)
```

Comment:
Preexisting, but just spotted this – support form should use a `NavigationStack` as `NavigationView` is deprecated.

Reply:
Good catch, updated on https://github.com/woocommerce/woocommerce-ios/pull/16313/commits/857b94853e16ff19c97effae98a46f1c0c8f1c0a 

I don't get warnings of this one, but seems we use it 75 more times across the app 😨 , will log it into the iOS17+ updates project